lib/cli/Command.ts

```diff
@@ -1,14 +1,20 @@
-import { Arguments } from 'yargs';
+import fs from 'fs';
+import path from 'path';
 import grpc from 'grpc';
+import errors from '../consts/Errors';
+import { Arguments } from 'yargs';
+import { getSystemHomeDir } from '../Utils';
 import { WalliClient } from '../proto/wallirpc_grpc_pb';
 
 interface GrpcResponse {
   toObject: Function;
 }
 
 export const loadWalliClient = (argv: Arguments): WalliClient => {
-  const credentials = grpc.credentials.createInsecure();
-  return new WalliClient(`${argv.rpc.host}:${argv.rpc.port}`, credentials);
+  const certPath = argv.tlscertpath ? argv.tlscertpath : path.join(getSystemHomeDir(), 'Walli', 'tls.cert');
```

Comment:
This will not work on any OS that is not Windows. Why not use the `getServiceDataDir` method?